src/bridging/BridgingSdk/BridgingSdk.ts

```diff
@@ -133,6 +133,7 @@ export class BridgingSdk {
         tradingSdk,
         provider: this.provider,
         getErc20Decimals,
+        bridgeHookSigner: advancedSettings?.quoteSigner,
```

Comment:
Custom signer for fetching quote with bridge hook because the fetch requires hooks signing and we don't want to ask users to sign anything while fetching quote